velox/substrait/VeloxToSubstraitType.cpp

```diff
@@ -158,11 +158,29 @@ const ::substrait::Type& VeloxToSubstraitTypeConvertor::toSubstraitType(
       substraitType->set_allocated_struct_(substraitStruct);
       break;
     }
+    case velox::TypeKind::DATE: {
```

Comment:
This is unrelated change, right? Please, extract it to a separate PR.

Reply:
done